src/ImageSharp/Formats/Webp/Lossless/LosslessUtils.cs

```diff
@@ -94,49 +94,53 @@ public static int PrefixEncode(int distance, ref int extraBits, ref int extraBit
     /// <param name="pixelData">The pixel data to apply the transformation.</param>
     public static void AddGreenToBlueAndRed(Span<uint> pixelData)
     {
-        if (Avx2.IsSupported)
+        if (Avx2.IsSupported && pixelData.Length >= 8)
```

Comment:
Did the length check early and made a `do-while`-loop.
This prevents the creation* of the vector `addGreenToBlueAndRedMaskAvx2` when not needed. 
The same on other loops in this file.

\* actually it's a memory load from the data-segment in recent .NET versions